### DISCUSSION TITLE
Why some hydra jobs are aborted?

### DISCUSSION THREAD

**Discussion by jian-lin** [nix-community/infra#1353] [Q&A] [answered]
228 jobs are aborted in https://hydra.nix-community.org/eval/510 .  I would like to know the causes and improve that.

**Comment by Mic92** [comment]
Usually one can manually abort builds from past evaluations. I suspect this is what happened here as well.

**Comment by jian-lin** [comment]
All evaluations listed [here](https://hydra.nix-community.org/jobset/emacs-overlay/master) have about 200 aborted jobs.  Is that normal?

**Comment by zowoq** [comment, marked as answer]
We're started building a new project on hydra and I've also been testing some unreleased hydra changes.

The aborted builds should have already been previously pushed to cachix but I've restarted them anyway.

**Comment by jian-lin** [comment]
> We're started building a new project on hydra

I am interested in this "new hydra" project.  Is there a way for me to contribute?

**Comment by zowoq** [comment]
We're building cuda and rocm package sets aren't built by hydra.nixos.org.

https://hydra.nix-community.org/project/nixpkgs
https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/release-cuda.nix

**Comment by jian-lin** [comment]
Oh, I misread.  I though you were implementing a new hydra.
